config/routes.rb

```diff
@@ -10,9 +10,18 @@
 
   resources :dashboard, only: [:index]
 
+  get '/cart', :to => 'carts#index', :as => 'cart'
 
-resources :items, only: [:index, :show]
+  resources :items, only: [:index, :show]
 
-resources :carts, only: [:index, :create, :destroy]
-get '/:category', to: 'categories#show', param: :slug, as: "category"
+  resources :carts, only: [:index, :create, :destroy]
```

Comment:
I think we might eventually want to change the index to show, and the resource to cart; only one can exist at a time, so it feels more like a singular use case to me.